src/script/lua_api/l_item.cpp

```diff
@@ -525,6 +525,29 @@ int ModApiItemMod::l_register_item_raw(lua_State *L)
 	return 0; /* number of results */
 }
 
+// unregister_item(name)
+int ModApiItemMod::l_unregister_item_raw(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	std::string name = luaL_checkstring(L, 1);
+
+	// Get the writable item definition manager from the server
+	IWritableItemDefManager *idef =
+			getServer(L)->getWritableItemDefManager();
+
+	// Unregister the node
```

Comment:
These comments are unhelpful.  They just repeat the name of the function being called.  Please remove.